guard/lib/guard/organization_cleaner.ex

```diff
@@ -0,0 +1,29 @@
+defmodule Guard.OrganizationCleaner do
+  use Quantum, otp_app: :guard
+
+  alias Guard.Store.Organization
+
+  require Logger
+
+  @chunk_size 10
+
+  def process do
+    Organization.find_candidates_for_hard_destroy()
+    |> Stream.chunk_every(@chunk_size)
+    |> Stream.each(fn orgs ->
+      orgs
+      |> Task.async_stream(
+        fn org ->
+          Logger.info("Hard destroying organization #{org.id}")
+          Organization.hard_destroy(org)
+        end,
+        max_concurrency: @chunk_size,
```

Comment:
This line is redundant. Each chunk has at max @chunk_size elements, Task.async_stream() has no way to start more than @chunk_size tasks, even without this specification